### PR TITLE
ADSECGH-33-Class-member-ordering-according-to-google-style-guide 

### PR DESCRIPTION
**Pull request by tlmnrnhrdt**

### CODE REVIEW

File: AdSecGH/AdSecGHInfo.cs [hunk 1]
```diff
@@ -142,112 +129,88 @@ private static string TryFindPluginPath(string keyword)
...
+    internal const string PluginName = "AdSecGH";
+    internal const string ProductName = "AdSec";
+    internal const string TermsConditions = "Oasys terms and conditions apply. See https://www.oasys-software.com/terms-conditions for details. ";
+    internal const string Vers = "0.9.17";
+    internal static string Disclaimer = PluginName + " is pre-release and under active development, including further testing to be undertaken. It is provided \"as-is\" and you bear the risk of using it. Future versions may contain breaking changes. Any files, results, or other types of output information created using " + PluginName + " should not be relied upon without thorough and independent checking. ";
+    internal static Guid GUID = new Guid("f815c29a-e1eb-4ca6-9e56-0554777ff9c9");
+    internal static bool isBeta = true;
+  }
+
+  internal sealed class PluginInfo {
```
Comment (DominikaLos) [inline AdSecGH/AdSecGHInfo.cs:202]: nested classes shouldn't be at the beginning?

File: AdSecGH/Components/1_Properties/CreateConcreteCrackParameters.cs [hunk 2]
```diff
@@ -12,128 +12,109 @@
...
 
     public CreateConcreteCrackParameters() : base(
-      "Create CrackCalcParams",
-      "CrackCalcParams",
-      "Create Concrete Crack Calculation Parameters for AdSec Material",
-      CategoryName.Name(),
-      SubCategoryName.Cat1())
-    {
-      this.Hidden = true; // sets the initial state of the component to hidden
+  "Create CrackCalcParams",
```
Comment (DominikaLos) [inline AdSecGH/Components/1_Properties/CreateConcreteCrackParameters.cs:26]: broken indentation

File: AdSecGH/Components/1_Properties/CreateCustomMaterial.cs [hunk 3]
```diff
@@ -14,33 +14,77 @@
...
 
     public CreateCustomMaterial() : base(
-      "Custom Material",
-      "CustomMaterial",
-      "Create a custom AdSec Material",
-      CategoryName.Name(),
-      SubCategoryName.Cat1())
-    {
-      this.Hidden = true; // sets the initial state of the component to hidden
+  "Custom Material",
```
Comment (DominikaLos) [inline AdSecGH/Components/1_Properties/CreateCustomMaterial.cs:28]: as above

File: AdSecGH/Components/1_Properties/CreateDesignCode.cs [hunk 4]
```diff
@@ -25,255 +22,234 @@ public class CreateDesignCode : GH_OasysDropDownComponent
...
-        "Another level",
-        "Another other",
-        "This is so deep"
-      });
-
-      this._dropDownItems = new List<List<string>>();
-      this._selectedItems = new List<string>();
-
-      List<string> designCodeGroups = Helpers.ReflectAdSecAPI.ReflectAdSecNamespace("Oasys.AdSec.DesignCode").Keys.ToList();
+      SubCategoryName.Cat1()) { Hidden = true; } // sets the initial state of the component to hidden
```
Comment (DominikaLos) [inline AdSecGH/Components/1_Properties/CreateDesignCode.cs:25]: object init another than in the rest of classes

File: AdSecGH/Components/4_Loads/CreateDeformation.cs [hunk 5]
```diff
@@ -1,121 +1,105 @@
...
+    // This region handles how the component in displayed on the ribbon including name, exposure level and icon
+    public override Guid ComponentGuid => new Guid("cbab2b58-2a01-4f05-ba24-2c79827c7415");
+    public override GH_Exposure Exposure => GH_Exposure.primary;
+    public override OasysPluginInfo PluginInfo => AdSecGH.PluginInfo.Instance;
+    protected override System.Drawing.Bitmap Icon => Properties.Resources.DeformationLoad;
+    private CurvatureUnit _curvatureUnit = DefaultUnits.CurvatureUnit;
+    private StrainUnit _strainUnit = DefaultUnits.StrainUnitResult;
+
+    public CreateDeformation() : base(
+  "Create Deformation Load",
```
Comment (DominikaLos) [inline AdSecGH/Components/4_Loads/CreateDeformation.cs:26]: broken indentation

File: AdSecGH/Components/Lab/FindCrackLoad.cs [hunk 6]
```diff
@@ -12,31 +12,24 @@
...
+  public class FindCrackLoad : GH_OasysComponent {
     // This region handles how the component in displayed on the ribbon including name, exposure level and icon
     public override Guid ComponentGuid => new Guid("f0b27be7-f367-4a2c-b90c-3ba0f66ae584");
     public override GH_Exposure Exposure => GH_Exposure.quarternary | GH_Exposure.obscure;
-    protected override System.Drawing.Bitmap Icon => Properties.Resources.CrackLoad;
     public override OasysPluginInfo PluginInfo => AdSecGH.PluginInfo.Instance;
+    protected override System.Drawing.Bitmap Icon => Properties.Resources.CrackLoad;
+
+    public FindCrackLoad() : base(
+  "Find Crack Load",
```
Comment (DominikaLos) [inline AdSecGH/Components/Lab/FindCrackLoad.cs:24]: as above

File: AdSecGH/Helpers/GH/RibbonLayout.cs [hunk 7]
```diff
@@ -1,76 +1,64 @@
...
-    public static string Cat8()
-    {
-      return "Lab";
-    }
-
-    public static string Cat9()
-    {
-      return "Params";
-    }
+  internal class SubCategoryName {
```
Comment (DominikaLos) [inline AdSecGH/Helpers/GH/RibbonLayout.cs:22]: first nested classes then fields right?

File: AdSecGH/Helpers/_ResultsHelper.cs [hunk 8]
```diff
@@ -61,144 +67,105 @@ internal GridAxis(float x_min, float x_max)
...
+      }
 
       return magMsd * magPow;
     }
   }
-  class Result
-  {
-    internal static List<double> SmartRounder(double max, double min)
-    {
+
```
Comment (DominikaLos) [inline AdSecGH/Helpers/_ResultsHelper.cs:117]: nested classes at the beginning right?

File: AdSecGH/Parameters/AdSecDeformationGoo.cs [hunk 9]
```diff
@@ -1,38 +1,31 @@
-﻿using System;
-using System.Linq;
-using Grasshopper.Kernel.Types;
+﻿using Grasshopper.Kernel.Types;
 using Oasys.AdSec;
-using OasysGH.Units;
+using OasysGH.Units;
 using OasysUnits;
+using System;
```
Comment (DominikaLos) [inline AdSecGH/Parameters/AdSecDeformationGoo.cs:5]: system at the beginning

**Review by DominikaLos — CHANGES_REQUESTED** [review]

**Review by kpne — APPROVED** [review]

File: AdSecGH/AdSecGHInfo.cs (hunk 1 above)
Comment (tlmnrnhrdt) [reply AdSecGH/AdSecGHInfo.cs:177]: it´s not nested ;)

Comment (tlmnrnhrdt) [reply AdSecGH/AdSecGHInfo.cs:177]: we should probably make it it´s own file..

File: AdSecGH/Helpers/GH/RibbonLayout.cs (hunk 7 above)
Comment (DominikaLos) [reply AdSecGH/Helpers/GH/RibbonLayout.cs:22]: it's not nested so I moved the second class to new file

File: AdSecGH/Helpers/_ResultsHelper.cs (hunk 8 above)
Comment (DominikaLos) [reply AdSecGH/Helpers/_ResultsHelper.cs:117]: also not nested, so  I moved it to new file

**Review by DominikaLos — APPROVED** [review]

**Pull request merged**